### DISCUSSION TITLE
Twitter API availability change

### DISCUSSION THREAD

**Discussion by manikmagar** [jreleaser/jreleaser#1194] [General]
Today Twitter made an announcement related to their API availability - https://twitter.com/TwitterDev/status/1621026986784337922?s=20&t=nxrqUIt8ZwmBii59P5zyjw. The Twitter4J uses Twitter API v1.1. Looks like this announcement may start causing jreleaser announcer failures. Just in case anyone wonders why the failures. 

![image](https://user-images.githubusercontent.com/877286/216502645-1ccccc33-0373-4e57-9f7a-e9c8e35137c8.png)

**Comment by aalmiray** [comment]
Indeed. We'll keep the Twitter announcer in place for those that'll continue to  use the API.
JReleaser itself will stop posting announcements to Twitter, opting for Mastodon -> https://fosstodon.org/@jreleaser

**Reply by manikmagar** [reply]
Yeah, that sounds right.
